### PR TITLE
* Update to 5.0.2

### PR DESCRIPTION
**Pull request by ag-ramachandran**

### CODE REVIEW

**Comment by ag-ramachandran** [comment]
Test Results

**Manual tests :** 

[E2E Test Runs.docx](https://github.com/Azure/logstash-output-kusto/files/12656261/E2E.Test.Runs.docx)

**Automated JRuby tests :** 

[e2e-run-logs.log](https://github.com/Azure/logstash-output-kusto/files/12656260/e2e-run-logs.log)


** Unit tests **

[unit-tests.log](https://github.com/Azure/logstash-output-kusto/files/12656289/unit-tests.log)

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -10,9 +10,9 @@ jobs:
     strategy:
       matrix:
         logstash: [
-          { version: '7.17.4', url: "https://artifacts.elastic.co/downloads/logstash/logstash-7.17.4-linux-x86_64.tar.gz" , main: 'true' },
-          { version: '6.8.23', url: "https://artifacts.elastic.co/downloads/logstash/logstash-6.8.23.tar.gz" , main: 'false' },
-          { version: '8.2.2', url: "https://artifacts.elastic.co/downloads/logstash/logstash-8.2.2-linux-x86_64.tar.gz" , main: 'false' }
+          #{ version: '7.17.4', url: "https://artifacts.elastic.co/downloads/logstash/logstash-7.17.4-linux-x86_64.tar.gz" , main: 'true' },
```
Comment (AsafMah) [inline .github/workflows/build.yml:13]: We are no longer building on the old versions?

File: CHANGELOG.md
Comment (AsafMah) [inline CHANGELOG.md:1]: Shouldn't this file be updated?

File: start-run.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+chmod -R 777 /src/logstash-output-kusto/*
```
Comment (AsafMah) [inline start-run.sh:2]: 777 is probably not what we want to expose

File: .github/workflows/build.yml (hunk 1 above)
Comment (ag-ramachandran) [reply .github/workflows/build.yml:13]: The latest version seems to be 8.10. Challenges with old LS versions is that they use outdated Jackson and other dependent libraries causing failures in installation and runs

Comment (AsafMah) [reply .github/workflows/build.yml:13]: Instead of commenting out, just remove everything not supported.

File: CHANGELOG.md [hunk 3]
```diff
@@ -34,4 +34,9 @@
 # 1.0.5
 
 - Use (3.1.3) version of the java sdk, and retrieve it from maven with bundler.
-- Added support for `proxy_host` `proxy_port` `proxy_protocol` to support proxying ingestion to Kusto
\ No newline at end of file
+- Added support for `proxy_host` `proxy_port` `proxy_protocol` to support proxying ingestion to Kusto
+
+# 2.0.0
```
Comment (AsafMah) [inline CHANGELOG.md:39]: Put new releases above old ones.

File: README.md [hunk 4]
```diff
@@ -71,6 +71,14 @@ export  LS_JAVA_OPTS="-Dhttp.proxyHost=1.2.34 -Dhttp.proxyPort=8989 -Dhttps.prox
 ```
 
 
+### Release Notes and versions
+
+| Version | Release Date | Notes |
+| --- | --- | --- |
+| 2.0.0 | 2023-09-19 | - Upgrade to the latest Java SDK version [5.0.2](https://github.com/Azure/azure-kusto-java/releases/tag/v5.0.2). Tests have been performed on **__Logstash 8.6__**  and up - Fixes CVE's in common-text & outdated Jackson libraries  |
```
Comment (AsafMah) [inline README.md:78]: Make it more clear that version 2.0 does not support logstash 6 & 7, and if they want to use it they have to use version 1.x.x

**Comment by ag-ramachandran** [comment]
superseded

**Pull request closed without merging**